src/ToolingNuget/Nuget/Microsoft.Windows.VbsEnclave.CodeGenerator.nuspec

```diff
@@ -27,8 +27,17 @@
     <file src="..\..\ToolingNuget\Nuget\Microsoft.Windows.VbsEnclave.CodeGenerator.props" target="build\native"/>
     <file src="..\..\ToolingNuget\Nuget\Microsoft.Windows.VbsEnclave.CodeGenerator.targets" target="build\native"/>
     <file src="..\..\ToolingSharedLibrary\Includes\VbsEnclaveABI\**" target="src\VbsEnclaveABI\" />
-    <file src="$vbsenclave_codegen_cpp_support_x64_lib$" target="lib\native\x64"/>
-    <file src="$vbsenclave_codegen_cpp_support_arm64_lib$" target="lib\native\arm64"/>
+
+    <!-- veil C++ support static lib -->
+    <file src="$vbsenclave_codegen_cpp_support_x64_Release_lib$" target="lib\native\x64"/>
+    <file src="$vbsenclave_codegen_cpp_support_ARM64_Release_lib$" target="lib\native\arm64"/>
+    <file src="$vbsenclave_codegen_cpp_support_x64_Debug_lib$" target="lib\native\x64"/>
+    <file src="$vbsenclave_codegen_cpp_support_ARM64_Debug_lib$" target="lib\native\arm64"/>
+    <file src="$vbsenclave_codegen_cpp_support_x64_Release_pdb$" target="lib\native\x64"/>
+    <file src="$vbsenclave_codegen_cpp_support_ARM64_Release_pdb$" target="lib\native\arm64"/>
+    <file src="$vbsenclave_codegen_cpp_support_x64_Debug_pdb$" target="lib\native\x64"/>
+    <file src="$vbsenclave_codegen_cpp_support_ARM64_Debug_pdb$" target="lib\native\arm64"/>
```

Comment:
i would have preferred to have a Debug/Release folder, and exclude the "ARM64_Release" part from the lib names.

(Also why is "ARM64" or "x64" in the name?)